decidim-debates/app/controllers/decidim/debates/admin/debates_controller.rb

```diff
@@ -7,7 +7,7 @@ module Admin
       class DebatesController < Decidim::Debates::Admin::ApplicationController
         helper Decidim::ApplicationHelper
 
-        helper_method :debates
+        helper_method :debates, :debate
```

Comment:
This `helper_method` isn't doing anything, as it's not used in the view.
```suggestion
        helper_method :debates
```

Reply:
Ok